### ISSUE TITLE
Switch to ndn-cxx segment fetcher

### ISSUE THREAD

**Issue by pulsejet** [named-data/ndn-svs#13]
This may need changes to the spec, but is much more maintainable.

**Comment by yoursunny** [comment]
> may need changes to the spec

You should never change a protocol to fit an implementation.
Instead, extend the segment fetcher to support the protocol.

**Comment by Pesa** [comment]
> You should never change a protocol to fit an implementation.

I don't think that's what's happening here. SegmentFetcher embodies a protocol (or convention) that is not specific to one implementation. My interpretation of this issue is simply to adjust the SVS spec as needed to conform to the SegmentFetcher conventions, which makes a lot of sense if we're talking about segmented objects.

**Comment by pulsejet** [comment]
Yeah, the current spec is rather simplistic and patches multi-segment support on top of the previous protocol. So it is probably worth investigating a more consistent naming convention with a new protocol version.
